src/IonTextWriter.ts

```diff
@@ -54,6 +54,8 @@ export class Context {
     }
 }
 
+let _jsbiZero = JSBI.BigInt(0);
```

Comment:
A constant for this is [available in in `JsbiSupport`](https://github.com/amzn/ion-js/blob/fa1fe7a6a8590d9fc5647f3a123495e7bad2a4d3/src/JsbiSupport.ts#L22).

Reply:
thank you :)